src/Component/HomePage.js

```diff
@@ -1,14 +1,35 @@
-import React from 'react';
+import React from "react";
 import logo from "../images/Split-it-logo.png";
+import SignUpModal from "./SignUpModal";
+import SignInModal from "./SignInModal";
 
-function HomePage() {
-    return (
-        <div>
-            <img src={logo} alt="Spli-it app logo" />
-            <button>Log-in</button>
-            <button>Sign-up</button>
-        </div>
-    )
+class HomePage extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			isModalOpen: false
+        };
+        this.handleModalWindow = this.handleModalWindow.bind(this);
+    }
+      handleModalWindow = e => {
+        e.preventDefault();
+        this.setState(() => ({ isModalOpen: !this.state.isModalOpen }));
+      };
+	render() {
+		return (
+			<div>
+				<img src={logo} alt="Spli-it app logo" />
+				<button onClick={this.handleModalWindow}>Log-in</button>
+				<button onClick={this.handleModalWindow}>Sign-up</button>
+                <SignUpModal 
+                openModal={this.state.isModalOpen} 
+                closeModal={this.handleModalWindow} />
+                <SignInModal   
```

Comment:
The 2 modals depend on the same state so once you open one of them the other one will open too.
Is better to have separate states for each modal.

Reply:
That's a great Catch Cristi!! Added separate state for both.